config.json.dist

```diff
@@ -0,0 +1,21 @@
+{
+    "name": "GUI",
+    "uiBaseUrl": "https://trello.com",
+    "apiBaseUri": "https://api.trello.com/1",
+    "accounts": [
+      {
+        "user": "user1",
+        "userName": "",
+        "password": "",
+        "apiKey": "",
+        "apiToken": ""
+      },
+      {
+        "user": "user2",
+        "userName": "",
+        "password": "",
+        "apiKey": "",
+        "apiToken": ""
+      }
+    ]
+}
```

Comment:
Add brackets to the json structure so a List can be received on the environment class methods.